sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -241,20 +241,14 @@ page at http://checkstyle.sourceforge.net/config.html -->
 
     -->
 
+    <!--
```

Comment:
Same comment as above: if you plan on merging this, can you add a comment explaining why it is necessary, and a tracking JIRA to re-enable it?

Reply:
Done